components/Gallery.tsx

```diff
@@ -0,0 +1,67 @@
+import { LinearGradient } from "expo-linear-gradient";
+import React from "react";
+import { Image, StyleSheet } from "react-native";
+import PagerView from "react-native-pager-view";
+import { View, Text } from "./Themed";
```

Comment:
Can we get an alphabetization on these imports please

Reply:
^ lol yeah that'd be a good practice